build-ps/debian/percona-server-server.postinst

```diff
@@ -116,9 +116,30 @@ case "$1" in
 				echo "!includedir /etc/mysql/percona-server.conf.d/" >> /etc/mysql/mysql.cnf
 			fi
 		fi
-                if [ "${CNF_ACTION}" = "Use NEW my.cnf" ]; then
-		        update-alternatives --force --install /etc/mysql/my.cnf my.cnf "/etc/mysql/mysql.cnf" 200
-                fi
+		if [ "${CNF_ACTION}" = "Use NEW my.cnf" ]; then
+			update-alternatives --force --install /etc/mysql/my.cnf my.cnf "/etc/mysql/mysql.cnf" 200
+		fi
+
+		PROFILE_ACTION="Use NEW AppArmor profile"
+		# If the existing AppArmor module/local profile is the proper file, we back it up
+		if [ -f "/etc/apparmor.d/usr.sbin.mysqld" -o -f "/etc/apparmor.d/local/usr.sbin.mysqld" ]; then
+			cp /etc/apparmor.d/usr.sbin.mysqld /etc/apparmor.d/usr.sbin.mysqld.bak 2> /dev/null || true
+			cp /etc/apparmor.d/local/usr.sbin.mysqld /etc/apparmor.d/local/usr.sbin.mysqld.bak 2> /dev/null || true
+			db_input high percona-server-server/existing_profile_file || true
```

Comment:
as we discussed for pxc - it is better to create a backup if the $RET is "Use NEW AppArmor profile"

Reply:
Fixed. I forgot that we have the same part here :)